src/Compiler/FSComp.txt

```diff
@@ -1491,7 +1491,6 @@ notAFunctionButMaybeIndexerWithName2,"This value is not a function and cannot be
 notAFunctionButMaybeIndexer2,"This expression is not a function and cannot be applied. Did you intend to access the indexer via 'expr[index]'?"
 3217,notAFunctionButMaybeIndexerErrorCode,""
 notAFunctionButMaybeDeclaration,"This value is not a function and cannot be applied. Did you forget to terminate a declaration?"
-3218,ArgumentsInSigAndImplMismatch,"The argument names in the signature '%s' and implementation '%s' do not match. The argument name from the signature file will be used. This may cause problems when debugging or profiling."
```

Comment:
I am not sure it's a good idea to remove the existing warning identifier, someone tooling (like codefixes) can be relying on it. What's the reasoin of moving it to just strings?

Reply:
Hey this will indeed likely break a code fix, the `RenameParamToMatchSignature` one, [here](https://github.com/dotnet/fsharp/blob/main/vsintegration/src/FSharp.Editor/CodeFixes/RenameParamToMatchSignature.fs). This should be addressed - which might be just changing the diag code in the fix/tests or something more elaborate.

Should be easy since we have [tests](https://github.com/dotnet/fsharp/blob/main/vsintegration/tests/FSharp.Editor.Tests/CodeFixes/RenameParamToMatchSignatureTests.fs) for this now. But I can help with this if needed.

But yeah more importantly - why the removal is needed?